src/xenia/kernel/xam/xam_ui.cc

```diff
@@ -575,6 +575,23 @@ dword_result_t XamShowDeviceSelectorUI_entry(
     dword_t user_index, dword_t content_type, dword_t content_flags,
     qword_t total_requested, lpdword_t device_id_ptr,
     pointer_t<XAM_OVERLAPPED> overlapped) {
+  X_RESULT error = X_ERROR_SUCCESS;
+
+  if ((user_index >= 4 && user_index != 0xFF) ||
+      (content_flags & 0x83F00008) != 0 || !device_id_ptr || !overlapped) {
+    error = X_ERROR_INVALID_PARAMETER;
```

Comment:
if you apply comment above then here you can remove that !overlapped check and just write:
`XOverlappedSetExtendedError(overlapped, X_ERROR_INVALID_PARAMETER);
 return X_ERROR_INVALID_PARAMETER;`